website/routes.py

```diff
@@ -1412,6 +1412,16 @@ def make_url_map(app):
         ),
 
         Rule(
+            [
+                '/project/<pid>/file_subscriptions/',
+                '/project/<pid>/node/<nid>/file_subscriptions/',
+            ],
+            'get',
```

Comment:
If there's no time they're going to be accessed, then remove them. If you were to keep them, I'd go with /project/{pid}/subscriptions/files/ as the endpoint (presumable there will be other types of subscriptions that go with the node ultimately)